app/test_split_integer.py

```diff
@@ -2,20 +2,35 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value = 15
+    number_of_parts = 5
+    result = split_integer(value, number_of_parts)
+    assert sum(result) == value
```

Comment:
```suggestion
    assert sum(split_integer(15, 5)) == 15
```
I think that it is not necessary to create a variable if it is used only once

Reply:
fix everywhere